theano/tensor/elemwise.py

```diff
@@ -502,6 +503,7 @@ class Elemwise(OpenMPOp):
     | Elemwise(log)(rand(3, 4, 5))
 
     """
+    __props__ = ("scalar_op", "inplace_pattern", "name", "nfunc_spec", "openmp")
```

Comment:
\__props__ don't support dict, (event when ignoring the caching). So don't add it here until we fix the general \__props__ mechanism to support that.